digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/AssetValidatorProcess.scala

```diff
@@ -0,0 +1,200 @@
+import fi.liikennevirasto.digiroad2.dao.{InaccurateAssetDAO, Queries}
+import fi.liikennevirasto.digiroad2.dao.linearasset.OracleSpeedLimitDao
+import fi.liikennevirasto.digiroad2.oracle.OracleDatabase
+import fi.liikennevirasto.digiroad2.process.{AssetServiceValidator, _}
+import fi.liikennevirasto.digiroad2.service.RoadLinkService
+import fi.liikennevirasto.digiroad2.service.pointasset.TrafficSignService
+import fi.liikennevirasto.digiroad2.user.UserProvider
+import org.joda.time.DateTime
+
+object AssetValidatorProcess {
+
+  lazy val properties: Properties = {
+    val props = new Properties()
+    props.load(getClass.getResourceAsStream("/bonecp.properties"))
+    props
+  }
+
+  lazy val dr2properties: Properties = {
+    val props = new Properties()
+    props.load(getClass.getResourceAsStream("/digiroad2.properties"))
+    props
+  }
+
+  lazy val vvhClient: VVHClient = {
+    new VVHClient(dr2properties.getProperty("digiroad2.VVHRestApiEndPoint"))
+  }
+
+  lazy val userProvider: UserProvider = {
+    Class.forName(dr2properties.getProperty("digiroad2.userProvider")).newInstance().asInstanceOf[UserProvider]
+  }
+
+  lazy val roadLinkService : RoadLinkService = {
+    new RoadLinkService(vvhClient, new DummyEventBus, new DummySerializer)
+  }
+
+  lazy val trafficSignService: TrafficSignService = {
+    new TrafficSignService(roadLinkService, userProvider, new DummyEventBus)
+  }
+
+  lazy val manoeuvreServiceValidator: ManoeuvreValidator = {
+    new ManoeuvreValidator()
+  }
+
+  lazy val hazmatTransportProhibitionValidator: HazmatTransportProhibitionValidator = {
+    new HazmatTransportProhibitionValidator()
+  }
+
+  lazy val heightLimitValidator: HeightLimitValidator = {
+    new HeightLimitValidator()
+  }
+
+  lazy val totalWeightLimitValidator: TotalWeightLimitValidator = {
+    new TotalWeightLimitValidator()
+  }
+
+  lazy val trailerTruckWeightLimitValidator: TrailerTruckWeightLimitValidator = {
+    new TrailerTruckWeightLimitValidator()
+  }
+
+  lazy val axleWeightLimitValidator: AxleWeightLimitValidator = {
+    new AxleWeightLimitValidator()
+  }
+
+  lazy val bogieWeightLimitValidator: BogieWeightLimitValidator = {
+    new BogieWeightLimitValidator()
+  }
+
+  lazy val widthLimitValidator: WidthLimitValidator = {
+    new WidthLimitValidator()
+  }
+
+  lazy val lengthLimitValidator: LengthLimitValidator = {
+    new LengthLimitValidator()
+  }
+
+  lazy val speedLimitValidator: SpeedLimitValidator = {
+    new SpeedLimitValidator(trafficSignService)
+  }
+
+  lazy val inaccurateAssetDAO : InaccurateAssetDAO = {
+    new InaccurateAssetDAO()
+  }
+
+  def verifyInaccurateSpeedLimits(): Unit = {
+    println("Start inaccurate SpeedLimit verification\n")
+    println(DateTime.now())
+    println("")
+
+    val polygonTools: PolygonTools = new PolygonTools()
+    val dao = new OracleSpeedLimitDao(null, null)
+
+    //Expire all inaccuratedAssets
+    OracleDatabase.withDynTransaction {
```

Comment:
Maybe we need to review this transaction, in case of some problem on the verification of the Inaccurate asset we already delete all inaccurate assets from the table.
If possible we need to make all the process together, if one fails, fail for all and return everything.